### PR TITLE
Add support for Kannada language (https://kn.wikipedia.org)

### PR DESCRIPTION
**Pull request by shivahr**
Thanks @lintool for this awesome code. Found exactly what I was looking for.

### CODE REVIEW

File: src/main/java/org/wikiclean/WikiClean.java [hunk 1]
```diff
@@ -131,7 +131,13 @@ public String getWikiMarkup(String s) {
       return "";
     }
 
-    return s.substring(textStart + 27, textEnd);
+    String s2 = s.substring(textStart + 27, textEnd);
```
Comment (shivahr) [inline src/main/java/org/wikiclean/WikiClean.java]: The XML files exported from https://en.wikipedia.org/wiki/Special:Export have an additional attribute `bytes` in `text` element. Need to ignore this as well while reading wiki-markup.

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@
 .project
 target/
 *~
+.DS_Store
```
Comment (shivahr) [inline .gitignore]: .DS_Store file on Mac needs to be ignored.

**Comment by lintool** [comment]
@shivahr Thanks for the contribution. Do you want me to cut another release and push to Maven central?

**Comment by shivahr** [comment]
Thanks for merging. And yes, pushing a new version to Maven central would certainly help.

**Comment by lintool** [comment]
Done! Thanks once again:
https://github.com/lintool/wikiclean/blob/master/HISTORY.md#version-11

**Comment by shivahr** [comment]
OK cool. Thank you!

**Pull request merged**